### PR TITLE
Shadow copy 7.0 not experimental

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/release-notes/aspnetcore-6.0.md [hunk 1]
```diff
@@ -670,7 +670,9 @@ For more information, see [StackExchange.Redis Profiling](https://stackexchange.
 
 ### Shadow copying in IIS
 
-An experimental feature has been added to the <xref:host-and-deploy/aspnet-core-module> to add support for [shadow copying application assemblies](/dotnet/framework/app-domains/shadow-copy-assemblies). Currently .NET locks application binaries when running on Windows making it impossible to replace binaries when the app is running. While our recommendation remains to use an [app offline file](xref:host-and-deploy/iis/app-offline), we recognize there are certain scenarios (for example FTP deployments) where it isn’t possible to do so.
+Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](host-and-deploy/iis/advanced?view=aspnetcore-7.0#shadow-copy).
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-6.0.md:673]:
```suggestion
Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true #shadow-copy).
```

File: aspnetcore/release-notes/aspnetcore-6.0.md [hunk 2]
```diff
@@ -670,7 +670,9 @@ For more information, see [StackExchange.Redis Profiling](https://stackexchange.
 
 ### Shadow copying in IIS
 
-An experimental feature has been added to the <xref:host-and-deploy/aspnet-core-module> to add support for [shadow copying application assemblies](/dotnet/framework/app-domains/shadow-copy-assemblies). Currently .NET locks application binaries when running on Windows making it impossible to replace binaries when the app is running. While our recommendation remains to use an [app offline file](xref:host-and-deploy/iis/app-offline), we recognize there are certain scenarios (for example FTP deployments) where it isn’t possible to do so.
+Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true #shadow-copy).
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-6.0.md:673]:
```suggestion
Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```

**Comment by HaoK** [comment]
@ldillonel @adityamandaleeka should we wait until 7.0 fully ships before updating 6.0 docs to say its officially supported?

**Comment by Rick-Anderson** [comment]
> @ldillonel @adityamandaleeka should we wait until 7.0 fully ships before updating 6.0 docs to say its officially supported?

I can change it to:

Shadow copy will be fully supported when .NET 7 releases.

File: aspnetcore/release-notes/aspnetcore-6.0.md [hunk 3]
```diff
@@ -670,7 +670,9 @@ For more information, see [StackExchange.Redis Profiling](https://stackexchange.
 
 ### Shadow copying in IIS
 
-An experimental feature has been added to the <xref:host-and-deploy/aspnet-core-module> to add support for [shadow copying application assemblies](/dotnet/framework/app-domains/shadow-copy-assemblies). Currently .NET locks application binaries when running on Windows making it impossible to replace binaries when the app is running. While our recommendation remains to use an [app offline file](xref:host-and-deploy/iis/app-offline), we recognize there are certain scenarios (for example FTP deployments) where it isn’t possible to do so.
+Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-6.0.md:673]:
```suggestion
Shadow copy is in preview now but will be supported when ASP.NET Core 7.0 is released. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```

File: aspnetcore/release-notes/aspnetcore-6.0.md [hunk 4]
```diff
@@ -670,7 +670,9 @@ For more information, see [StackExchange.Redis Profiling](https://stackexchange.
 
 ### Shadow copying in IIS
 
-An experimental feature has been added to the <xref:host-and-deploy/aspnet-core-module> to add support for [shadow copying application assemblies](/dotnet/framework/app-domains/shadow-copy-assemblies). Currently .NET locks application binaries when running on Windows making it impossible to replace binaries when the app is running. While our recommendation remains to use an [app offline file](xref:host-and-deploy/iis/app-offline), we recognize there are certain scenarios (for example FTP deployments) where it isn’t possible to do so.
+Shadow copy is now in preview but will be supported when ASP.NET Core 7.0 is released. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-6.0.md:673]: @ldillonel @adityamandaleeka  I updated it to say 
Shadow copy is now in preview but will be supported when ASP.NET Core 7.0 is released.
I could that to
Shadow copy is now in preview but **is expected*** to be supported when ASP.NET Core 7.0 is released.

**Pull request closed without merging**